sharepoint/sharepoint-server-ps/sharepoint-server/Set-SPWebApplication.md

```diff
@@ -354,29 +457,99 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -DisableSMTPEncryption
-{{Fill DisableSMTPEncryption Description}}
+### -SMTPServer
+Specifies the new outbound SMTP server that this Web application will use.
+Set to $null to clear the server setting.
 
 ```yaml
-Type: SwitchParameter
+Type: String
 Parameter Sets: UpdateMailSettings
 Aliases: 
 Applicable: SharePoint Server Subscription Edition
 
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SMTPServerPort
+Specifies a SMTP port value.
+
+```yaml
+Type: Int32
+Parameter Sets: UpdateMailSettings
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Url
+Specifies the load-balanced URL for the Web application zone.
+
+```yaml
+Type: String
+Parameter Sets: UpdateIISSettings
+Aliases:
+Applicable: SharePoint Server Subscription Edition
+
 Required: False
 Position: Named
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -NotProvisionGlobally
-{{Fill NotProvisionGlobally Description}}
+### -UseServerNameIndication
+Specifies that the Secure Sockets Layer (SSL) binding of this IIS web site should use Server Name Indication (SNI).
```

Comment:
This should be "Secure Socket Layer" rather than plural.

Reply:
Actually, it should be plural "Secure Sockets Layer" rather than singular.  (I know, but that's what it's officially called...)

Reply:
@tseward @TroyStarr So we would keep "Secure Sockets Layer" instead of changing.